### ISSUE TITLE
Questions about table 4

### ISSUE THREAD

**Issue by ax7e** [ByteDance-Seed/Seed-Thinking-v1.5#3]
In this table, Doubao-150B-MoE shows superior performance compared to Qwen-32B. Could this be interpreted as evidence that Doubao-150B-MoE is a better base model than Qwen-32B?

Specifically, I'm wondering if factors such as datasets, training steps, learning rates, and other reinforcement learning algorithms were controlled in this experiment to ensure a fair comparison between the models.
